src/core/dialer.c

```diff
@@ -414,9 +414,9 @@ nni_dialer_start(nni_dialer *d, unsigned flags)
 	int      rv = 0;
 	nni_aio *aio;
 
-	if (nni_atomic_flag_test_and_set(&d->d_started)) {
-		return (NNG_ESTATE);
-	}
+//	if (nni_atomic_flag_test_and_set(&d->d_started)) {
+//		return (NNG_ESTATE);
+//	}
```

Comment:
You can try to remove these comments if you have rebased the latest master branch